<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dodawanie</class>
 <widget class="QDialog" name="Dodawanie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="dodawaniLatwePage">
      <widget class="QPushButton" name="sprawdzButton">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>380</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Sprawdz wynik</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="punktyEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>350</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="punktyLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>350</y>
         <width>37</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Punkty:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="plusEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>140</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="skladnikPierwszyEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="skladnikDrogiEdit">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>130</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="rownasieEdit">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>140</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sumaEdit">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>140</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="zyciaEdit">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>30</y>
         <width>104</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="zycieLabel">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Życia:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="menuPoziomTrudnosciDodawaniePage">
      <widget class="QPushButton" name="latweButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>381</width>
         <height>141</height>
        </rect>
       </property>
       <property name="text">
        <string>Łatwe</string>
       </property>
      </widget>
      <widget class="QPushButton" name="srednieButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>190</y>
         <width>371</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>Srednie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="trudneButton_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>310</y>
         <width>361</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>Trudne</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>430</y>
         <width>361</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>Bardzo trudne</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="dodatanieTrudnePage">
      <widget class="QTextEdit" name="zyciaEdit_2">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>20</y>
         <width>104</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="zycieLabel_2">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Życia:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="skladnikDrogiEdit_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>130</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="sumaEdit_2">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>130</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sprawdzButton_2">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>370</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Sprawdz wynik</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="plusEdit_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>130</y>
         <width>21</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="punktyEdit_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>340</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="skladnikPierwszyEdit_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="rownasieEdit_2">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>130</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
      <widget class="QLabel" name="punktyLabel_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>340</y>
         <width>37</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Punkty:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="skladnikTrzeciEdit_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>130</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="plusDwaEdit_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>130</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
